lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -444,7 +444,7 @@ namespace Js
                 inlineeOffset = inlinedFrameWalker.GetCurrentInlineeOffset();
             }
         }
-        else if (ScriptFunction::Is(parentFunction) && HasInlinedFramesOnStack())
+        else if (ScriptFunction::Test(parentFunction) && HasInlinedFramesOnStack())
```

Comment:
Let me correct, you need to only change to `ScriptFunction::Test` at call sites that is guarded by `JavascriptFunction::Is()`